lib/judge_core/src/logic/problem_presets/normal_judge.rs

```diff
@@ -1,15 +1,164 @@
-use crate::model::*;
+use crate::constant::label::single_judge;
+use crate::constant::*;
+use crate::logic::procedure_builder::ProcedureBuilder;
+use crate::model::{procedure::writer_schema::*, *};
 
 pub struct NormalJudgeTestcase {
     pub name: String,
     pub input: String,
     pub expected_output: String,
 }
 
+static NJ_BUILD_SCRIPT: &str = include_str!("./normal_judge_build.py");
+static NJ_RUN_SCRIPT: &str = include_str!("./normal_judge_run.py");
+static NJ_SUMMARIZE_SCRIPT: &str = include_str!("./normal_judge_summarize.py");
+
+static COMPILE_PHASE_TIME_RESERVED_MS: i64 = 30000;
+static TEST_PHASE_TIME_RESERVED_MS: i64 = 6000;
```

Comment:
```suggestion
static TEST_PHASE_TIME_RESERVED_MS: i64 = 11000;
```
1ケース10秒の問題とかあるし11秒の方が良さそう